lib/matplotlib/transforms.py

```diff
@@ -653,30 +657,62 @@ def rotated(self, radians):
         return bbox
 
     @staticmethod
-    def union(bboxes):
+    def union(bboxes, null_as_empty=True):
         """Return a `Bbox` that contains all of the given *bboxes*."""
         if not len(bboxes):
             raise ValueError("'bboxes' cannot be empty")
         # needed for 1.14.4 < numpy_version < 1.16
         # can remove once we are at numpy >= 1.16
         with np.errstate(invalid='ignore'):
-            x0 = np.min([bbox.xmin for bbox in bboxes])
-            x1 = np.max([bbox.xmax for bbox in bboxes])
-            y0 = np.min([bbox.ymin for bbox in bboxes])
-            y1 = np.max([bbox.ymax for bbox in bboxes])
+            x0 = np.min([bbox.x0 for bbox in bboxes])
+            x1 = np.max([bbox.x1 for bbox in bboxes])
+            y0 = np.min([bbox.y0 for bbox in bboxes])
+            y1 = np.max([bbox.y1 for bbox in bboxes])
+        if not null_as_empty:
+            cbook.warn_deprecated(
+                3.4, message="Bboxs will soon change their behavior to "
+                "correctly treat empty Bboxs as empty sets in unions and "
```

Comment:
Struggling with this error message 'cause on the one hand like I know why you're writing it cause the operations are unions & intersections...and on the other it's meaningless w/o an understanding of basic set theory. What concretely/what input is triggering this error?"